$NetBSD: patch-fsnotify.cabal,v 1.1 2022/02/26 12:33:45 pho Exp $

Use hinotify on some BSDs. Not upstreamed yet; can only be upstreamed after
../../devel/hs-hinotify/patches/patch-hinotify.cabal gets merged.

--- fsnotify.cabal.orig	2022-02-26 11:04:55.313630845 +0000
+++ fsnotify.cabal
@@ -38,7 +38,7 @@ Library
                         , System.FSNotify.Types
   Hs-Source-Dirs:       src
   GHC-Options:          -Wall
-  if os(linux)
+  if os(linux) || os(freebsd) || os(netbsd) || os(openbsd)
     CPP-Options:        -DOS_Linux
     Other-Modules:      System.FSNotify.Linux
     Build-Depends:      hinotify >= 0.3.0,
